src/libproxychains.c

```diff
@@ -517,8 +529,15 @@ struct hostent *gethostbyaddr(const void *addr, socklen_t len, int type) {
 #   define MSG_FASTOPEN 0x20000000
 #endif
 
+#if defined HAVE_GCC_DIAGNOSTIC
+# pragma GCC diagnostic push
+# pragma GCC diagnostic ignored "-Wpedantic"
+#endif
 ssize_t sendto(int sockfd, const void *buf, size_t len, int flags,
 	       const struct sockaddr *dest_addr, socklen_t addrlen) {
+#if defined HAVE_GCC_DIAGNOSTIC
+# pragma GCC diagnostic pop
+#endif
```

Comment:
sorry, but it is not acceptable to litter the code with ifdefs and nasty compiler-specific pragma just to avoid warnings that shouldnt be used.